package.json

```diff
@@ -37,7 +37,8 @@
   "scripts": {
     "fmt": "semistandard --fix",
     "lint": "semistandard --verbose | snazzy",
-    "test": "npm run lint && mocha test"
+    "test": "npm run lint && mocha test",
+    "cov": "istanbul cover ./node_modules/.bin/_mocha test"
```

Comment:
I add a `cov` command, do you want to run coverage in travis-ci? we use https://codecov.io, and it's good.